Services/Interfaces/web_interface/models/interface_settings.py

```diff
@@ -21,15 +21,16 @@
 
 def add_watched_symbol(symbol):
     watched_symbols = models.get_watched_symbols()
-    watched_symbols.append(symbol)
-    return _save_edition()
+    if symbol not in watched_symbols:
+        watched_symbols.append(symbol)
+        return _save_edition()
```

Comment:
Sorry, I just checked and we also need to add `return True` at the end of the method since the return value is important here. Same in `remove_watched_symbol`.